sql/codegen.go

```diff
@@ -134,6 +134,28 @@ func newFiller(pr *extendedSelect, fts fieldTypes, db *DB) (*filler, error) {
 				r.FeatureColumnsCode[target],
 				feaColCode)
 		}
+		for _, spec := range colSpecs {
```

Comment:
One thing to discuss is, whether we need to involve `featureSpec` in `codegen.go`, in `codegen_alps.go` `featureSpec` and `featureColumn` are both needed to:

- `featureSpec`: Used to generate code for `input_fn` and dataset
- `featureColumn`: Used to generate `feature_column` code to pass to `Estimators`

for one column, `DENSE(col)` and `NUMERIC(col)` must both appear to generate correct code, and it's a bit hard to use.

So I'd like to consider use only one structure to store all column's meta information.

Reply:
> So I'd like to consider use only one structure to store all column's meta information.

You mean to create a struct called `denseColumn`?

Reply:
Done.